x/savings/keeper/deposit_test.go

```diff
@@ -0,0 +1,149 @@
+package keeper_test
+
+import (
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/crypto"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmtime "github.com/tendermint/tendermint/types/time"
+
+	"github.com/kava-labs/kava/app"
+	"github.com/kava-labs/kava/x/savings/types"
+)
+
+func (suite *KeeperTestSuite) TestDeposit() {
+	type args struct {
+		depositor                 sdk.AccAddress
+		amount                    sdk.Coins
```

Comment:
would call this `depositAmount` and add `initialDepositorBalance` as an arg (see below)

Reply:
Updated in latest